pavelib/utils/db_utils.py

```diff
@@ -12,7 +12,7 @@
 from pavelib.utils.envs import Env
 
 CACHE_FOLDER = 'common/test/db_cache'
-FINGERPRINT_FILEPATH = '{}/{}/bokchoy_migrations.sha1'.format(Env.REPO_ROOT, CACHE_FOLDER)
+FINGERPRINT_FILEPATH = '{}/{}/bok_choy_migrations.sha1'.format(Env.REPO_ROOT, CACHE_FOLDER)
```

Comment:
I'm curious why this changed, but I'm not familiar with this code so I'm probably missing some context.

Reply:
The names in this directory were inconsistent and bothering me: https://github.com/edx/edx-platform/tree/master/common/test/db_cache. The sha1 file is only referenced in this variable, so changing it won't break anything.